### ISSUE TITLE
Arbiter: Missing configuration options

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#6062]
Thanks to @eraly 

- BatchNormalizationSpace does not support constrainBeta and constrainGamma options
- ConvolutionLayerSpace, SubsamplingLayerSpace and Deconvolution2DLayerSpace do not support kernel dilation and use the default value of {1,1}
- Currently EmbeddingLayerSpace does not support hasBias and uses the default value of False
- Currently OCNNLayerSpace do not support the configureR (uses default value of true), initialRValue (uses default value of 0.1) and windowSize. What is referred to hiddenLayerSize in the OCNNOutputLayer.Builder is referred to as numHidden in OCNNLayerSpace.Builder.
- Currently SeparableConvolution2DLayerSpace does not support constrainPointWise

**Comment by eraly** [comment]
So looking at the dl4j code it looks like the  constrainBeta and constrainGamma options get used in BatchNormalizationSpace. It looks like it is getting used in model import for keras layers. I put in the skeleton for it anyway and we can fix it in dl4j later?

It gets really messy on instantiating discrete layer constraints parameter space is...Any thoughts to clean that up?
https://github.com/deeplearning4j/deeplearning4j/blob/a7cf56fee0dec7b5e9404e528a4cbb86946a9c89/arbiter/arbiter-deeplearning4j/src/test/java/org/deeplearning4j/arbiter/multilayernetwork/TestLayerSpace.java#L131-L137

**Comment by AlexDBlack** [comment]
Yeah, constrainBeta/Gamma aren't a big deal, but we might as wel fix it in DL4J later.

As for the constraints parameter space code there - I'd just do this instead, which is probably good enough for now:
https://gist.github.com/AlexDBlack/bdcfce752a812007c3d9e40e0af5ebe0

**Comment by eraly** [comment]
Ah cool. I'll switch to that in the test. Thanks.
I'll open an issue for constrainBeta/Gamma in a bit.

**Comment by AlexDBlack** [comment]
https://github.com/deeplearning4j/deeplearning4j/pull/6089
